### ISSUE TITLE
diagnose: If not all channels are locked, warnings are not shown

### ISSUE THREAD

**Issue by PsikoBlock** [totev/vodafone-station-cli#142]
When running the diagnose command, I currently have some channels displaying as NotLocked. The power level in the corresponding row is displayed as NaN. Consequently, an error message is shown:
```
Error: PowerLevel is not within supported range. PowerLevel: NaN
```
Instead, I would like to see the warning messages (if any) about the remaining channels, e.g., 
```
Legend: pl = power level | snr = signal to noise ration
Colors:  Fix within one Month | Fix immediately
DOWN: ch193pl, ch29snr, ch193snr
UP:
```
My setup:
- Arris TG3442DE
- `vodafone-station-cli/1.2.8 linux-x64 node-v19.9.0`
- Arch Linux

I've attached the full output: [vodafone-station-cli-diagnose.txt](https://github.com/totev/vodafone-station-cli/files/11303214/vodafone-station-cli-diagnose.txt)

**Comment by totev** [comment]
Thanks for the issue report!
Do you happen to know what the original value coming from the modem is in that field? Maybe by executing it with `env DEBUG=\* ./bin/dev diagnose`?
The current expectation is, that the value is formatted like so `"PowerLevel":"-2.8\/57.2"`.

**Comment by totev** [comment]
@PsikoBlock If you can, could you please check the fix here: #143 ?

**Comment by totev** [comment]
Potentially fixed with #143
